core/src/main/java/com/sequenceiq/cloudbreak/service/decorator/responseprovider/StackResponseEventProvider.java

```diff
@@ -15,17 +19,34 @@
 
 @Component
 public class StackResponseEventProvider implements ResponseProvider {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(StackResponseEventProvider.class);
+
     @Inject
     private CloudbreakEventsFacade cloudbreakEventsFacade;
 
     @Override
     public StackV4Response providerEntriesToStackResponse(Stack stack, StackV4Response stackResponse) {
+        List<CloudbreakEventV4Response> events = new ArrayList<>();
         List<CloudbreakEventV4Response> cloudbreakEvents = cloudbreakEventsFacade
                 .retrieveEventsByStack(stack.getId(), stack.getType(), PageRequest.of(0, Integer.MAX_VALUE)).getContent();
-        stackResponse.setCloudbreakEvents(cloudbreakEvents);
+        events.addAll(cloudbreakEvents);
+        events.addAll(getLegacyStackType(stack));
+        stackResponse.setCloudbreakEvents(events);
         return stackResponse;
     }
 
+    private List<CloudbreakEventV4Response> getLegacyStackType(Stack stack) {
```

Comment:
I don't see any tests which ensure that events are not broken again.

Reply:
I think we don't have any tests what can catch this. Here is the use case:
Start a distrox with 2.24
Update Cb to 2.25
Before the fix, the resource type with stacks was filtered out. 

If you want I can write a UT, but I think this test will not prevent similar cases.